### PR TITLE
fix(component): Fix EMPTY_MESSAGE appearing in web when media message sent from android

### PR DESCRIPTION
**Pull request by fauzanlinnas**
> This project is a forked version of the [Sendbird UI Kit](https://github.com/sendbird/sendbird-uikit-react) to match with the Ruangguru's internal requirements. Therefore, it is not yet set up to accept pull requests from external contributors. But you can always freely create a forked version from this repository to match your own requirements.

## Related Issue

[RKLS-1088](https://ruanggguru.atlassian.net/browse/RKLS-1088)

## Description Of Changes

Fix EMPTY_MESSAGE appearing in web when media message sent from android

### Before
Screen Shot 2021-11-09 at 14 11 36

### After
Screen Shot 2021-11-09 at 14 11 22

### CODE REVIEW

**Comment by gunturvo** [comment]
test comment

File: src/smart-components/Conversation/components/MessageHOC.jsx [hunk 1]
```diff
@@ -11,8 +11,9 @@ import MessageContent from '../../../ui/MessageContent';
 import DateSeparator from '../../../ui/DateSeparator';
 import Label, { LabelTypography, LabelColors } from '../../../ui/Label';
 import MessageInput from '../../../ui/MessageInput';
-import FileViewer from '../../../ui/FileViewer';
-import RemoveMessageModal from './RemoveMessage';
+import FileViewer from '../../../rogu/ui/FileViewer';
```
Comment (suliskh) [inline src/smart-components/Conversation/components/MessageHOC.jsx:14]: Mas @fauzanlinnas, is this file supposed to be edited?

File: src/smart-components/App/stories/index.stories.js [hunk 2]
```diff
@@ -11,6 +11,8 @@ import MessageSearch from '../../MessageSearch';
 import { withSendBird } from '../../..';
 import { sendBirdSelectors } from '../../..';
 import { fitPageSize } from './utils';
+import COLOR_SET from "../../../../__mocks__/themeMock";
```
Comment (suliskh) [inline src/smart-components/App/stories/index.stories.js:14]: Mas @fauzanlinnas, is this file supposed to be edited?

File: src/smart-components/App/index.jsx [hunk 3]
```diff
@@ -9,7 +9,7 @@ import PropTypes from 'prop-types';
 import Sendbird from '../../lib/Sendbird';
 
 import ChannelList from '../ChannelList';
-import Conversation from '../Conversation';
+import Conversation from '../../rogu/smart-components/Conversation';
```
Comment (suliskh) [inline src/smart-components/App/index.jsx:12]: Mas @fauzanlinnas, is this file supposed to be edited?

File: src/ui/MessageContent/index.tsx [hunk 4]
```diff
@@ -80,7 +80,7 @@ export default function MessageContent({
   const [supposedHover, setSupposedHover] = useState(false);
 
   const isByMe: boolean = isPendingMessage(channel, message as UserMessage | FileMessage)
-    || isSentMessage(channel, message as UserMessage | FileMessage)
+    || !isSentMessage(channel, message as UserMessage | FileMessage)
```
Comment (suliskh) [inline src/ui/MessageContent/index.tsx:83]: Mas @fauzanlinnas, is this file supposed to be edited?

**Review by suliskh — CHANGES_REQUESTED** [review]
Mas @fauzanlinnas, ada beberapa file changes yg sepertinya unexpected. Minta tolong dicek lagi yah mas

File: src/ui/MessageContent/index.tsx (hunk 4 above)
Comment (fauzanlinnas) [reply src/ui/MessageContent/index.tsx:83]: lah iya ini file2nya ga sengaja ke push juga
wait mas

**Pull request closed without merging**